src/software/thunderscope/field/field.py

```diff
@@ -7,12 +7,33 @@ class Field(pg.PlotWidget):
 
     """Wrapper to handle Field Layers"""
 
+    MAX_X_RANGE = 10000
+    MAX_Y_RANGE = 6000
```

Comment:
would be nice if this were dynamic with div a vs div b

Reply:
yeah the automatic thing kind off breaks it,

I changed it to passing in these values into the constructor, so we can set this from thunderscope.py when the time comes